util/find-doc-nits

```diff
@@ -1029,21 +1026,43 @@ my %skips = (
     'digest' => 1,
 );
 
+my %genopts; # generic options parsed from apps/include/opt.h
+
 # Check the flags of a command and see if everything is in the manpage
 sub checkflags {
     my $cmd = shift;
     my $doc = shift;
-    my %cmdopts;
+    my @cmdopts;
     my %docopts;
     my %localskips;
 
-    # Get the list of options in the command.
-    open CFH, "$openssl list --options $cmd|"
-        or die "Can list options for $cmd, $!";
+    # Get the list of options in the command source file.
+    my $src = $cmd;
+    $src = "asn1pars" if $cmd eq "asn1parse";
+    $src = "crl2p7" if $cmd eq "crl2pkcs7";
```

Comment:
Just a thought: why not rename the source file to match the command name?

Reply:
Good idea. The short file names were due to DOS with a maximum of 8 chars in the file name. Those days are past, so there's no reason to keep that adaptation.
I assume that's a PR away 😉

Reply:
Better here.

Reply:
I was also already considering renaming the two source files, which is the most clean solution.
Done.